pages/legal/licences.md

```diff
@@ -46,6 +46,8 @@ La deuxième partie de cette page dresse la liste des licences spéciales homolo
 | [GNU Lesser General Public License v3.0 or later](https://spdx.org/licenses/LGPL-3.0-or-later.html#licenseText) | `LGPL-3.0-or-later` |
 | [GNU Affero General Public License v3.0 or later](https://spdx.org/licenses/AGPL-3.0-or-later.html#licenseText) | `AGPL-3.0-or-later` |
 | [Mozilla Public License 2.0](https://spdx.org/licenses/MPL-2.0.html#licenseText)                                | `MPL-2.0`           |
+| [Eclipse Public License 2.0](https://spdx.org/licenses/EPL-2.0.html#licenseText)                                | `EPL-2.0`           |
+| [European Union Public License 2.0](https://spdx.org/licenses/EUPL-1.2.html#licenseText)                        | `EUPL-2.0`          |

```

Comment:
It seems there is a typo here.  
AFAIK, the latest version of the EUPL is 1.2. The url is correct but the text indicates 2.0.
I will open a PR.

Reply:
Thanks for spotting and reporting this, I just opened a PR.